src/settings.yml

```diff
@@ -0,0 +1,44 @@
+networks:
```

Comment:
pls remove any networks that aren't being tested

Reply:
Resolved